src/index.ts

```diff
@@ -310,6 +317,14 @@ class Spanner extends GrpcService {
       packageJson: require('../../package.json'),
     } as {} as GrpcServiceConfig;
     super(config, options);
+
+    if (
```

Comment:
Why not replace with `routeToLeaderEnabled===false`

Reply:
`routeToLeaderEnabled` will be assigned `false` based on SpannerOptions which we are doing in this condition.

Reply:
My bad I meant why not check `options.routeToLeaderEnabled===false`
From what I understand in case options.routeToLeaderEnabled is undefined then it won't be equal to false. And the other condition checks for if it is true